saleor/attribute/models/base.py

```diff
@@ -214,9 +214,9 @@ def get_translated_keys(self):
 
 
 class AttributeValue(SortableModel, ModelWithExternalReference):
-    name = models.CharField(max_length=250)
+    name = models.CharField(max_length=255)
```

Comment:
The name should be shorter than a slug. 
We generate a slug from the name and add a suffix to the slug in case of a duplicate. 
If someone uses a name with 255 chars suffix exceed a maximum number of characters in the slug field. 
Sufix generator - `saleor/saleor/core/utils/__init__.py::generate_unique_slug`

Reply:
Maybe I could change the function so that it cuts the name and adds a suffix?

If I make the name shorter than a slug it will be inconsistent with the other models' fields and that's what we are aiming at here.